src/wiki/plugins/inputs/settings.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.conf import settings as django_settings
+
+SLUG = 'inputs'
+
+INPUTS = getattr(
+    django_settings,
+    'WIKI_PLUGINS_INPUTS',
+    ('text', 'text_inline',
+     'password', 'password_inline',
```

Comment:
I don't think asking for a password with no secure storage mechanism is a good idea.